lib/src/boilerplate_suggestors/stubbed_props_and_state_class_remover.dart

```diff
@@ -18,15 +18,21 @@ import 'package:over_react_codemod/src/boilerplate_suggestors/advanced_props_and
 import 'package:over_react_codemod/src/boilerplate_suggestors/simple_props_and_state_class_migrator.dart';
 import 'package:over_react_codemod/src/dart2_suggestors/props_and_state_companion_class_remover.dart';
 
+import 'boilerplate_utilities.dart';
+
 /// Suggestor that removes every companion class for props and state classes, as
 /// they were only temporarily required for backwards-compatibility with Dart 1.
 class StubbedPropsAndStateClassRemover
     extends PropsAndStateCompanionClassRemover implements Suggestor {
+  final SemverHelper semverHelper;
```

Comment:
This is necessary because the current implementation of `StubbedPropsAndStateClassRemover` uses utility functions that require `SemverHelper` to be passed in. Once PR #82 (which updates the implementation and removes those utility function usages) is merged in, this will no longer be needed and can be removed.